### PR TITLE
docs: User guide outline changes for devworkspace only release

### PR DESCRIPTION
**Pull request by themr0c**
## What does this pull request change

doc: User guide outline changes for devworkspace only release

1. Identify documentation subject to removal by commenting out sections in the outline. Validation by @l0rd @benoitf. Reach consensus.
2. Cleanup content. Delete obsolete files. Handle redirections.

## What issues does this pull request fix or reference

fixes https://github.com/eclipse/che/issues/21273

## Pull request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/end-user-guide/nav.adoc [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
-** xref:defining-custom-commands-for-che-theia.adoc[]
-** xref:version-control.adoc[]
-** xref:che-theia-troubleshooting.adoc[]
-** xref:differences-in-how-che-theia-webview-works-on-a-single-host-mode-comparing-to-a-multi-host-mode.adoc[]
+// * xref:navigating-che.adoc[]
+// ** xref:navigating-che-using-the-dashboard.adoc[] // NOT NECESSARY
+// ** xref:importing-certificates-to-browsers.adoc[] // OBSOLETE
+* xref:accessing-che-from-openshift-developer-perspective.adoc[]
+// * xref:che-theia-ide-basics.adoc[] // REDIRECT TO https://theia-ide.org/docs/
+// ** xref:defining-custom-commands-for-che-theia.adoc[] // REDIRECT TO https://devfile.io/docs/devfile/2.0.0/user-guide/adding-commands-to-a-devfile/
```
Comment (nickboldt) [inline modules/end-user-guide/nav.adoc]: should this be to latest standard, https://devfile.io/docs/devfile/2.1.0/user-guide/adding-commands-to-a-devfile/ ?

File: modules/end-user-guide/nav.adoc [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
+// ** xref:version-control.adoc[] // REDIRECT TO https://theia-ide.org/docs/
+// ** xref:che-theia-troubleshooting.adoc[] // REMOVE
+// ** xref:differences-in-how-che-theia-webview-works-on-a-single-host-mode-comparing-to-a-multi-host-mode.adoc[] // REMOVE
 * xref:workspaces-overview.adoc[]
-** xref:creating-a-workspace-from-a-code-sample.adoc[]
+// ** xref:creating-a-workspace-from-a-code-sample.adoc[] // REMOVE USELESS
```
Comment (nickboldt) [inline modules/end-user-guide/nav.adoc]: https://github.com/eclipse-che/che-docs/blob/master/modules/end-user-guide/partials/assembly_creating-a-workspace-from-a-code-sample.adoc

Agreed, this is useless :)

File: modules/end-user-guide/nav.adoc [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
+// * xref:using-artifact-repositories-in-a-restricted-environment.adoc[] // TO REWRITE
+// ** xref:using-maven-artifact-repositories.adoc[] // TO REWRITE
+// ** xref:using-gradle-artifact-repositories.adoc[] // TO REWRITE
+// ** xref:using-python-artifact-repositories.adoc[] // TO REWRITE
+// ** xref:using-go-artifact-repositories.adoc[] // TO REWRITE 
+// ** xref:using-nuget-artifact-repositories.adoc[] // TO REWRITE
+// ** xref:using-npm-artifact-repositories.adoc[] // TO REWRITE
 * xref:troubleshooting-che.adoc[]
 ** xref:viewing-che-workspaces-logs.adoc[]
 ** xref:investigating-failures-at-a-workspace-start-using-the-verbose-mode.adoc[]
```
Comment (nickboldt) [inline modules/end-user-guide/nav.adoc:16]: https://github.com/eclipse-che/che-docs/blob/master/modules/end-user-guide/partials/proc_investigating-failures-at-a-workspace-start-using-the-verbose-mode.adoc

does this do anything other than opening the console log tab? when loading doesn't work, it's not very verbose... troubleshooting usually then involves going to the admin console, opening the pod, and reading ITS logs to see what went wrong. the dashboard doesn't expose enough of the pod log to be useful. 

@l0rd wondering if this section ought to be replaced (or enhanced) with content about how to find your workspace's pod in the admin console and then debug via that... ?

File: modules/end-user-guide/nav.adoc [hunk 4]
```diff
@@ -1,49 +1,49 @@
...
+* xref:accessing-che-from-openshift-developer-perspective.adoc[]
+// * xref:che-theia-ide-basics.adoc[] // REDIRECT TO https://theia-ide.org/docs/
+// ** xref:defining-custom-commands-for-che-theia.adoc[] // REDIRECT TO https://devfile.io/docs/devfile/2.0.0/user-guide/adding-commands-to-a-devfile/
+// ** xref:version-control.adoc[] // REDIRECT TO https://theia-ide.org/docs/
+// ** xref:che-theia-troubleshooting.adoc[] // REMOVE
+// ** xref:differences-in-how-che-theia-webview-works-on-a-single-host-mode-comparing-to-a-multi-host-mode.adoc[] // REMOVE
 * xref:workspaces-overview.adoc[]
-** xref:creating-a-workspace-from-a-code-sample.adoc[]
+// ** xref:creating-a-workspace-from-a-code-sample.adoc[] // REMOVE USELESS
 ** xref:creating-a-workspace-from-a-template-devfile.adoc[]
```
Comment (nickboldt) [inline modules/end-user-guide/nav.adoc]: I don't think this is still a viable option https://github.com/eclipse-che/che-docs/blob/master/modules/end-user-guide/partials/assembly_creating-a-workspace-from-a-template-devfile.adoc

AFAICT, there are two options: load from git, or open a sample:

image

File: modules/end-user-guide/nav.adoc (hunk 1 above)
Comment (themr0c) [reply modules/end-user-guide/nav.adoc]: Yes

File: modules/end-user-guide/nav.adoc (hunk 4 above)
Comment (themr0c) [reply modules/end-user-guide/nav.adoc]: Indeed

**Pull request merged**